page.md

```diff
@@ -0,0 +1,5 @@
+The pig was _____ years old.
+
+[this file](https://github.com/MaximBravo/create-your-own-adventure/one.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.